pages/common/snowsql.md

```diff
@@ -0,0 +1,24 @@
+# snowsql
+
+> SnowSQL command-line client for Snowflake's Data Cloud.
+> More information: <https://docs.snowflake.com/en/user-guide/snowsql.html>.
+
+- Connect to an instance at {{account}}.snowflakecomputing.com (password can be provided in prompt or configuration file):
+
+`snowsql --accountname {{account}} --username {{username}} --dbname {{database}} --schemaname {{schema}}`
+
+- Connect to an instance specified by the configuration file (by default, `~/.snowsql/config`):
+
+`snowsql --config {{.snowsql/config}}`
```

Comment:
```suggestion
`snowsql --config {{~/.snowsql/config}}`
```


- This change is optional - no corresponding style guide requirement exist.
- The corresponding change should be applied to `.pl` translation if it's used here.
- It's better to have identical paths.